src/startGame.js

```diff
@@ -0,0 +1,75 @@
+import Car from "./car.js";
+
+let carArray = [];
+let winner = [];
+
+function startGame(cars, num) {
+  // 자동차 객체 생성
+  makeCarObj(cars);
+  // 횟수만큼 레이싱 시키고 결과 저장하기
+  for(let i=0;i<num;i++) {
+    racing();
+    raceStatePrint();
+  }
+  // 우승자 선정
+  selectWinner();
+  // 우승자 출력
+  printWinner();
+}
+
+function makeCarObj(cars) {
+  for(let i in cars) {
+    let car = new Car(cars[i]);
+    carArray.push(car);
```

Comment:
이 친구도 마친가지로,
```suggestion
  cars.forEach((i) => {
     let car = new Car(i);
     carArray.push(car);
  })
```
이렇게 하시면 될 것 같습니다 👍